daprdocs/content/en/operations/hosting/kubernetes/kubernetes-annotations.md

```diff
@@ -35,3 +35,4 @@ The following table shows all the supported pod Spec annotations supported by Da
 | `dapr.io/sidecar-readiness-probe-period-seconds`  | How often (in seconds) to perform the sidecar readiness probe. Read more [here](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#configure-probes). Default is `6`
 | `dapr.io/sidecar-readiness-probe-threshold`       | When the sidecar readiness probe fails, Kubernetes will try N times before giving up. In  this case, the Pod will be marked Unready. Read more about `failureThreshold` [here](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#configure-probes). Default is `3`
 | `dapr.io/http-max-request-size`                   | Increasing max size of request body http and grpc servers parameter in MB to handle uploading of big files. Default is `4` MB      
+| `dapr.io/env`                                     | List of environment variable strings consisting of `key=value` pairs separated by a comma.
```

Comment:
Can we add to the description a sentence of what this does aside from the format required here? (as in other annotations)

Reply:
@orizohar
environment variables from the annotation are inejcted to sidecar. it is an appropriate sentence?

Reply:
Suggestion "List of environment variable *to be injected into the sidecar*. Strings consisting of `key=value` pairs separated by a comma